manager/state/raft/transport/transport.go

```diff
@@ -86,6 +86,7 @@ func (t *Transport) run(ctx context.Context) {
 		t.stopped = true
 		for _, p := range t.peers {
 			p.stop()
+			p.cc.Close()
```

Comment:
I don't know how do it properly :/ When peer is removed, I need to defer its closure, but here I need to close it immediately.